client/pages/recycle-and-reuse.js

```diff
@@ -10,107 +9,61 @@ import {
   TabPanel,
   Flex,
   Button,
-} from "@chakra-ui/react";
-import AddItems from "../components/recycleAndReuseComponents/AddItem";
-import VerifyItems from "../components/recycleAndReuseComponents/VerifyItems";
-import Geolocation from "../components/recycleAndReuseComponents/Geolocation";
-import axios from "axios";
-import React, { useState, useContext } from "react";
-import urlcat from "urlcat";
-import { options } from "../../mockData/data";
+} from '@chakra-ui/react'
 // STEPPER IMPORTS
-import { Step, Steps, useSteps } from 'chakra-ui-steps';
-// CONTEXT
-import FormContext from "../context/context";
-import next from "next";
+import { Step, Steps } from 'chakra-ui-steps'
 
-// // TODO: Need to actually fetch the data from the real server
-const hasNoItems = (items) => items?.input?.length === 0;
+import Additem from '../components/recycleAndReuseComponents/AddItem'
+import TakeAction from '../components/recycleAndReuseComponents/TakeAction'
+import VerifyItem from '../components/recycleAndReuseComponents/VerifyItem'
 
+import axios from 'axios'
+import { useState, useRef } from 'react'
+import urlcat from 'urlcat'
+
+// TODO: Need to actually fetch the data from the real server
 export async function getStaticProps() {
   //   const url = urlcat(process.env.SERVER_URL, "/api/items");
-  const url = urlcat("https://api.npoint.io", "a8416aa207861acb363d");
+  const url = urlcat("https://api.npoint.io", "324727202a8ce24af12f");
   try {
     const data = await axios.get(url);
     return {
       props: {
-        options: data.data,
+        data: data.data,
       },
     };
   } catch (error) {
-    return {
-      props: {
-        options,
-      },
-    };
+    // TODO: handle error in a better way
+    console.error(error)
   }
 }
 
-
-const steps = [
-  { id: 0, label: "Add Items" },
-  { id: 1, label: "Verify Items" },
-  { id: 2, label: "Take Action" },
-  { id: 3, label: "Complete!" }
-]
-
-function RecycleAndReuse({ options, pageProps }) {
-  const [selectedItems, setSelectedItems] = useState();
-  // const [items, setItems] = useState({ input: [] });
-
-  const { nextStep, prevStep, reset, activeStep } = useSteps({
-    initialStep: 0,
-  })
+function RecycleAndReuse({ data }) {
+  const [items, setItems] = useState([])
+  const [step, setStep] = useState(0)
 
   return (
-    <FormContext.Provider value={{ selectedItems, setSelectedItems }}>
```

Comment:
@CippyLOL Context is definitely a possible solution as well. However, since we are only tracking the items at the moment, let's keep things simple and just use the state + props drilling for children component.

Maybe in future if we have many children components it would be better to use context then